demos/http/common/src/http_demo_utils.c

```diff
@@ -31,6 +31,9 @@
 /* Retry utilities. */
```

Comment:
This file is copied from https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1318 and changes are the responsibility of that PR.